solutions/lamp-stack-vm-v1/azuredeploy-minimal.json

```diff
@@ -28,7 +28,7 @@
                     "CMSApplication":               { "value": "None" }
                 },
                 "templateLink": {
-                    "uri": "https://raw.githubusercontent.com/Azure/LAMP/master/azuredeploy.json"
+                    "uri": "https://raw.githubusercontent.com/Azure/solution-center/main/solutions/lamp-stack-vm-v1/azuredeploy.json"
```

Comment:
i think this also needs to be done for the other two lamp solutions?

Reply:
Changed